firewood/src/v2/emptydb.rs

```diff
@@ -146,4 +167,22 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn empty_streamer() -> Result<(), Error> {
+        let emptydb = EmptyDb {};
+        let rev = emptydb.revision(ROOT_HASH).await?;
+        let mut iter = rev.iter().await?;
+        assert!(iter.next().await.is_none());
+        Ok(())
+    }
+
+    #[tokio::test]
+    async fn empty_streamer_start_at() -> Result<(), Error> {
+        let emptydb = EmptyDb {};
+        let rev = emptydb.revision(ROOT_HASH).await?;
+        let mut iter = rev.iter_from(b"ignored").await?;
+        assert!(iter.next().await.is_none());
+        Ok(())
+    }
```

Comment:
These are both testing a hard-coded value... I'm not sure it's worth having these tests here unless there's actual functionality to test.